core/org.osate.core/src/org/osate/core/OsateCorePlugin.java

```diff
@@ -339,4 +344,27 @@ public final int getSOMLimit() {
 		final IPreferenceStore store = getPreferenceStore();
 		return store.getInt(MAX_SOM);
 	}
-}
\ No newline at end of file
+
+	/**
+	 * @since 2.1
+	 */
+	public final boolean getAlwaysShowInstantiationResults() {
+		final IPreferenceStore store = getPreferenceStore();
+		return store.getBoolean(ALWAYS_SHOW_INSTANTIATION_RESULTS);
+	}
+
+	/**
+	 * @since 2.1
+	 */
+	public static void setAlwaysShowInstantiationResultsDefault(final IPreferenceStore store) {
+		store.setDefault(OsateCorePlugin.ALWAYS_SHOW_INSTANTIATION_RESULTS,
+				OsateCorePlugin.ALWAYS_SHOW_INSTANTIATION_RESULTS_DEFAULT);
+	}
+
+	/**
+	 * @since 2.1
+	 */
+	public static String getAlwaysShowInstantiationResultsPreferenceName() {
+		return ALWAYS_SHOW_INSTANTIATION_RESULTS;
+	}
```

Comment:
I'd prefer to not have these two methods. IMO, they just make code more difficult to follow.